packages/medusa/src/services/__tests__/product-variant.js

```diff
@@ -718,33 +766,31 @@ describe("ProductVariantService", () => {
       expect(moneyAmountRepository.create).toHaveBeenCalledWith({
         variant_id: IdMap.getId("ironman"),
         region_id: IdMap.getId("cali"),
+        currency_code: "usd",
         amount: 100,
       })
 
-      expect(moneyAmountRepository.save).toHaveBeenCalledTimes(1)
+      expect(moneyAmountRepository.insertBulk).toHaveBeenCalledTimes(1)
     })
 
-    it("successfully creates a currency price", async () => {
+    it("successfully updates a currency price", async () => {
       await productVariantService.updateVariantPrices(IdMap.getId("ironman"), [
         {
           id: IdMap.getId("dkk"),
           currency_code: "dkk",
-          amount: 750,
+          amount: 850,
         },
       ])
 
       expect(moneyAmountRepository.create).toHaveBeenCalledTimes(0)
 
-      expect(
-        moneyAmountRepository.upsertVariantCurrencyPrice
-      ).toHaveBeenCalledTimes(1)
-      expect(
-        moneyAmountRepository.upsertVariantCurrencyPrice
-      ).toHaveBeenCalledWith(IdMap.getId("ironman"), {
-        id: IdMap.getId("dkk"),
-        currency_code: "dkk",
-        amount: 750,
-      })
+      expect(moneyAmountRepository.update).toHaveBeenCalledTimes(1)
+      expect(moneyAmountRepository.update).toHaveBeenCalledWith(
+        { id: IdMap.getId("dkk") },
+        {
+          amount: 850,
```

Comment:
self-review: Here only the amount changed, this is why the currecny_cde does not appear anymore